public/app/plugins/panel/canvas/module.tsx

```diff
@@ -22,6 +23,14 @@ export const addStandardCanvasEditorOptions = (builder: PanelOptionsEditorBuilde
     description: 'Enable selection of experimental element types',
     defaultValue: true,
   });
+
+  builder.addBooleanSwitch({
+    path: 'displayMiniMap',
+    name: 'Mini Map',
+    description: 'Display mini map during zoom',
+    defaultValue: false,
+    showIf: (opts) => config.featureToggles.canvasPanelPanZoom,
+  });
```

Comment:
After playing around with this locally I think it probably makes sense to include a toggle to enable / disable zoom / pan mode - in this current iteration I'm not sure the minimap really adds too much value so IMO we could axe it entirely in favor of a high level toggle for overall zoom / pan functionality. In my experience the minimap just ends up hiding content / has no interact-able functionality so maybe we should hold off on it

Reply:
Possible future feature request but being able to set the scale / save the scale in the panel model might be nice so that users can set a default zoom setting and have another way to modify the scaling (similar to geomap zoom settings 🤔?)

https://github.com/grafana/grafana/assets/22381771/f02909f0-b347-428f-be30-e8b7b05c752f